### ISSUE TITLE
Add discussions to the documents repository

### ISSUE THREAD

**Issue by roberthstrand** [open-gitops/documents#44]
Based on the conversation in the todays WG Fortnightly, did we want to add discussions to this repository and add categories based on work streams? We obviously still need to define how we want things documented (pun intended) but if we get a place where we can suggest specific topics that we want to explore or check for interest, we would be better situated when we start to work.

Of the top of my head the only category suggestions I could come up with is:
* Security
* Lifecycle (and/or) Operations
* Patterns

Having these discussions in this repository would help to keep things tidy, and not overwhelm people when they look at discussions in the project repo.

**Comment by lloydchang** [comment]
I don't have an opinion either way.

Your description about "project" repository versus "documents" repository makes sense to me, but that is an orthogonal problem.

❓ Are we conflating the orthogonal problems between "projects" usage and "documents" having "Issues" and "Discussions"?

You are saying that you want to add three new categories, 1. Security, 2. Lifecycle (and/or) Operations, 3. Patterns, as "Discussions" in this "documents" repository.

Issues tend to be sorted by labels, which require an administrator to label.

Discussions can change categories by the person who posted it.

That may be the important difference between "Issues" and "Discussions" — Removing repository administrators as a bottleneck to label each issue as 1. Security, 2. Lifecycle (and/or) Operations, 3. Patterns, and let the person who posted a discussion choose a category from 1. Security, 2. Lifecycle (and/or) Operations, 3. Patterns.

❓ Would those three be additional categories be additions to the defaults of Announcements, General, Ideas, Q&A, Show and tell?

These repositories, "project" and "documents", aren't that active relative to code repositories throughout GitHub.

❓ Are we trying to prematurely solve an organizational scaling problem that doesn't exist yet, and won't exist anytime soon?

A counterpoint to adding "Discussions" is that there is one less place to look for information in the "documents" repository, and less is more.

To recap, I don't have an opinion either way.

Thank you @roberthstrand

**Comment by scottrigby** [comment]
We have a single place for discussions on the project repo https://github.com/open-gitops/project/discussions

Agree searching the discussions for topics is a good idea! It seems to work pretty well now

Re work streams, we also have the TEAMS document to help with that https://github.com/open-gitops/project/pull/148 😄
